data/human/deep missions.txt

```diff
@@ -2431,3 +2431,320 @@ event "battle in arneb end"
+			`	After hours of hard work, the storm begins to subside and the sea and sky begin to calm. An exhausted Thorleif and Helm stagger over to you, and Thorleif surveys his small island. "Without your help I would have lost many of the outbuildings. It was a lucky thing you came when you did."`
+			`	With that he turns and heads back to the main dwelling, with you and Helm in tired tow.`
+			choice
+				`	(Wash up and warm up in the main room around the fire.)`
+					goto fire
+			label fire
+			`	 You spend the evening warming yourself by a roaring fire telling Helm and Thorleif tales of your exploits in space, and many of the wonders you've seen.`
+			`	In turn they tell you about the sea beasts, leviathans, and krakens that they have met on the ocean currents of Norn. Of lost isles filled with forgotten treasure or strange people, or stories even, about the ancient elves who once aided their ancestors and are said to be around still.`
+			choice
+				`	(Ask about the elves.)`
+					goto elves
+				`	(Ask about lost treasure.)`
+					goto treasure
+			label elves
+			`	The idea of Elves, or Alfar as they call them, intrigues you and you ask what they know of them.`
+			`	"The Alfar," Thorleif begins, "are said to have been a great help to us in the early days. Some say they spared us from plagues and protected us from monsters. Many still offer sacrifices and prayers to them, so that they remember us and know that we are friends and allies still."`
+			`	Thorleif shrugs, "This may be true, and good at that, but others think the Alfar, for all their apparent good works, cannot be trusted. Their minds are unknowable and filled with mischief, and their aims are not ours. This is the belief that our family holds to. They should be placated, but also feared. We must look to each other for support and strength, and trust not in the deeds of strangers."`
+					goto family
+			label treasure
+			`	You could always use a little extra money, and this trip, while engaging, has certainly not made you any richer, so you ask about hidden treasures. Helm and Thorleif both grin at this and glance at each other knowingly. They launch into tales of the riches of lost civilizations, or forgotten pirate hoards, or even early human kingdoms. It's easy to lose things on a fog-covered ocean world with floating islands, all too easy.`
+			`	When they are done regaling you with tales they mention to you that maybe some other time they'll take you out into the deep ocean and perhaps you'll find what you're looking for. They smile wolfishly at that and you think they might be putting you on. "Have you ever found treasure out here?" you ask.`
+			`	Thorleif shrugs, "coming here.. from Alfheim long ago, our descendants felt that the world itself was the real treasure. We have no need to go tracing dreams in the abyss."`
```

Comment:
```suggestion
			`	Thorleif shrugs. "Coming here... from Alfheim long ago, our descendants felt that the world itself was the real treasure. We have no need to go chasing dreams in the abyss."`
```
It feels like `chasing` might fit better than `tracing` here. Any particular reason you used `tracing`?

Reply:
No, I probably meant chasing and wrote tracing.